src/applications/apps.js

```diff
@@ -415,8 +415,15 @@ export function pathToActiveWhen(path, exactMatch) {
   const regex = toDynamicPathValidatorRegex(path, exactMatch);
 
   return (location) => {
+    // compatible with IE10
+    let origin = location.origin;
+    if (!origin) {
+      origin = `${location.protocol}//${location.hostname}${
```

Comment:
I think this could be simplified to the following, since `location.host` includes the port and is supported in IE10. See https://developer.mozilla.org/en-US/docs/Web/API/Location/host

```suggestion
      origin = `${location.protocol}//${location.host}`
```

Reply:
done